content/en/quickstart/_index.md

```diff
@@ -63,13 +65,9 @@ control plane version: 1.8.2-distro-v0
 data plane version: 1.8.2-distro-v0 (2 proxies)
```

Comment:
since this page is being updated, perhaps we should update the versions here as well

Reply:
replace the command output. Thanks!